R/copy.R

```diff
@@ -0,0 +1,32 @@
+#' Copy an entire object
+#'
+#' Creates a deep copy of an object, as an external pointer, for instance a
+#' [tokens_xptr] class object.
+#'
+#' @param x the input object to be copied
+#' @returns an entire copy of `x`, by value
+#' @export
+#' @examples
+#' xtoks <- tokens(data_corpus_inaugural[1:3], xptr = TRUE)
+#' print(xtoks, 0, 0)
+#' 
+#' # shallow copy
+#' xtoks_shallow <- xtoks
+#' print(xtoks_shallow, 0, 0)
+#' 
+#' # deep copy
+#' xtoks_deep <- copy(xtoks)
+#' print(xtoks_deep, 0, 0)
+copy <-  function(x) {
+    UseMethod("copy")
+}
+
+#' @export
+copy.default <- function(x) {
+    return(x)
+}
+
+#' @export
+copy.tokens_xptr <- function(x) {
+    as.tokens_xptr(x)
```

Comment:
Don't you think `as.tokens_xptr(xtoks)` is enough? For me, it makes sense to have `copy()` only if there are other objects that users need to deep copy.

Reply:
You always tried to avoid expanding the package names space.

Reply:
I think the semantics of `copy()` are easier and more consistent for the user than coercing each xptr object to its plain version, plus there is precedent in another package.

But it makes sense to me to avoid if it we are just talking about tokens for now. if we expand the xptr range, we can always add it back then. How about I just comment out this code for now?

Reply:
Removed in [d50727d](https://github.com/quanteda/quanteda/pull/2255/commits/d50727df08d027698f433256e81eec0abd102afc)